src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsauditapi/resource/AuditResourcePagingTest.kt

```diff
@@ -171,8 +186,10 @@ class AuditResourcePagingTest : IntegrationTest() {
 
     @Test
     fun `filter audit events by what and who`() {
-      webTestClient.get().uri("/audit/paged?page=0&size=3&what=OFFENDER_DELETED&who=bobby.beans")
+      webTestClient.post().uri("/audit/paged?page=0&size=3")
         .headers(setAuthorisation(roles = listOf("ROLE_AUDIT"), scopes = listOf("read")))
+        .contentType(MediaType.APPLICATION_JSON)
+        .bodyValue(whoAndWhatRequestBody.toString())
         .exchange()
         .expectStatus().isOk
         .expectBody()
```

Comment:
Needs further test cases for date and service filters, plus any validation failure cases

Reply:
I will look at that as well. Thanks.